### PR TITLE
Add ADX query walkthrough

### PR DESCRIPTION
**Pull request by baanders**
## Purpose

* Add ADX query examples and walkthrough

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[X] Other... Please describe: Sample code
```

### CODE REVIEW

File: adx-queries/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# ADX plugin sample queries and walkthrough
```
Comment (jjadta) [inline adx-queries/README.md:1]: ADT query plugin for ADX: Sample Queries and Walkthrough

Comment (jjadta) [reply adx-queries/README.md:1]: Please change any references of "ADX Query" as in the title to ADT Query Plugin for ADX: Sample Queries, or something similar to ensure the ADT/ADX reference.

File: adx-queries/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Click on one of the leaf nodes that has this model: `dtmi:example:grid:plants:domesticConsumer;1`.
+
+1. Update the PowerRequirement value to 500 and save it (using the disc button circled in red below) to patch the twin.
+
+    ![Screenshot of updating and saving a value in Azure Digital Twins Explorer.](../images/adx-queries/save-patch.png)
+
+## Step 2: Run joint Azure Digital Twins/ADX queries
+
+1. [Connect to the Samples database](https://dataexplorer.azure.com/clusters/help/databases/Samples) on the public ADX help cluster. The database features the table **SamplePowerRequirementHistorizedData** that contains historized property values over a one month period for selected digital twins in the graph.
```
Comment (jjadta) [inline adx-queries/README.md:56]: Please obscure the image where it shows the name of my ADT instance in the ADX explorer window

Comment (baanders) [reply adx-queries/README.md:56]: Removing your name and replacing with a generic sample name, ADTinstance (so readers can still see the format of the value from the screenshot)

**Pull request merged**